website/templates/prereg_landing_page.mako

```diff
@@ -23,16 +23,10 @@
 
 <%def name="existingPrereg(size=None)">
 <% size = size or '' %>
-<div id="existingPrereg${size}" class="prereg-exisitng-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none">
+<div id="existingPrereg${size}" class="prereg-exisitng-prereg p-md osf-box box-round clearfix m-b-lg" style="display:none; style="width: 100%;"">
```

Comment:
The mako variable is part of a previous change. I think changing this here would require some further code change and should probably be handled as a task of its own.